examples/07_advanced/plot_ica_neurovault.py

```diff
@@ -70,8 +69,8 @@
 print("\nReshaping and masking images.\n")
 
 with warnings.catch_warnings():
-    warnings.simplefilter("ignore", UserWarning)
-    warnings.simplefilter("ignore", DeprecationWarning)
+    # warnings.simplefilter("ignore", UserWarning)
```

Comment:
to be removed

Reply:
Quite a few UserWarning so will keep at least one (though I do not like to have examples that show the bad practice of ignoring warnings)